src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -22,7 +21,7 @@
 
     private final Optional<BibDatabase> database;
 
-    private final LayoutFormatter toUnicode = new LatexToUnicodeFormatter();
+    private final LatexToUnicodeFormatter toUnicode = new LatexToUnicodeFormatter();
```

Comment:
The problem was a name clash with `java.util.Formatter` which could be solved by importing `net.sf.jabref.logic.formatter.Formatter` explicitly. I've done that and will merge the PR when the checks are completed.